contributor_ladder.md

```diff
@@ -93,11 +93,9 @@ An Project Member must meet the responsibilites and has the requirements of a Co
 
 
 The process for a Contributor to become a Project Member is as follows:
-
-<!-- the process of becoming an organization member is going to depend strongly on how your project manages its infrastructure. TO DO: Project leads to fil in exact details of how a contributor becomes an organization member-->
-1.
-2.
-
+1. Community and/or project leadership recognizes significant project contributions made and sponsors a project contributor to be promoted
+2. A new project is on-boarded requiring one or more project members
+3. An OWNERS_ALIAS file at the project level will track roles which can be updated by submitting a PR from an existing project member
 
```

Comment:
hmmm, this isn't a step-by-step procedure that someone can follow.  This section is meant to be steps, not alternatives.

That is:  how do we want someone to apply for project member? An issue?  A PR?  Where? What happens after they get accepted (like, do they get added to certain GH teams?)

Reply:
So, an example of this would be:

1. The nominator will open an issue against the Community repository, titled "Candidate for Project Membership: NAME".  Candidates may self-nominate.
2. This issue will tag two current project members (sponsors) who can attest to the candidate's contributions.
3. Both sponsors will +1 the nomination.
4. A member of the Konveyor Github admin team will add the new project member to the Github organization and [list appropriate teams here]

Reply:
Ack. I was looking at this from a qualification standpoint vs. all out process.

Your suggestion makes sense. One change I would make is:

    ...Candidates may self-nominate. <<<---what happens if someone nominates themselves and is rejected? Are we opening a point of community contention? I would rather avoid that.

Reply:
It's not like the candidate wouldn't know they've been nominated, so they're going to know about a rejection regardless.  Also, keep in mind we're talking about org members here.  Rejections are either going to be based on stats ("you have zero contributions, please make a few and come back") or on trustworthiness ("you signed the CLA based on a company that doesn't exist") and neither thing is "political".

Discussions of Reviewer and Approver status have more reasons to be sometimes private, because those are places you need to evaluate quality.  For org member you really just want to know three things: (1) is this person making ongoing contributions? (2) are the trustworthy at some low level? (3) have they done the paperwork?  Really what Org Member is about is "the reviewers would like to stop having to /ok-to-test this person".

Reply:
The only serious problem I've seen with org membership nominations is when someone gets nominated for Org member based on their employer/job *before* having made any contributions, and doesn't want to wait.  This is one of the reasons we want to set the thresholds low; this will include Red Hat new hires, so we don't want them waiting a year to get their org status approved.  Please do examine the thresholds with that in mind, like do we actually want to shorten the required calendar time?

Reply:
@jberkus thank you for helping to explain, really helped me to understand what this role is for when you mentioned "the reviewers would like to stop having to /ok-to-test this person"